.github/workflows/develop.yml

```diff
@@ -116,6 +116,14 @@ jobs:
       - name: Start Apache
         run: service apache2 start
 
+      - name: Install memcache extension
+        run: |
+          apt-get -y update
+          apt-get -y install php5-memcache memcache
```

Comment:
Not sure if `php5-memcache` is required, since were testing with php 7.3.

Reply:
Trying the service approach